src/components/BookCard.js

```diff
@@ -1,6 +1,35 @@
 import React from "react";
-import BookCard from "./BookCard";
+import "./Book.css";
+import { Link } from "react-router-dom";
+import { Nav, Form, FormControl, Button, NavDropdown } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
 
-export default function BookCard() {
-  return <div></div>;
+export default function BookCard(props) {
```

Comment:
a bit nicer to not wrap everything in .data, and just pass that object directly?